featuretools/tests/entityset_tests/test_es.py

```diff
@@ -31,6 +31,46 @@ def test_normalize_time_index_as_additional_column(es):
                                copy_columns=[])
 
 
+def test_normalize_time_index_as_copy_column(es):
+    assert "signup_date" in es["customers"].columns
+    es.normalize_dataframe(base_dataframe_name='customers',
+                           new_dataframe_name='cancellations',
+                           index='cancel_reason',
+                           make_time_index='signup_date',
+                           additional_columns=[],
+                           copy_columns=['signup_date'])
+
+    assert 'signup_date' in es['cancellations'].columns
+    assert es['cancellations'].ww.time_index == 'signup_date'
```

Comment:
is this sufficient to check that we didn't accidentally move the column instead of copy it? does ww reset the time index if the time index column is removed?

Reply:
Ah, I see. If we add a check that `signup_date in es['customers']` and maybe a check that it's still the time index there as well, that will be a more clear check that it's being copied and not moved.

The answer to whether ww resets the time index if that column is removed is yes. The concept of what a "time index" is is reliant on a column that has a `time_index` tag, so if the column is not on the DataFrame, the DataFrame has no time index. But Featuretools probably shouldn't have to rely on that intimate a knowlege of ww.

Reply:
added checks to all of these tests that confirm that `signup_date` is still in the original dataframe